packages/app/src/cli/models/extensions/extension-instance.ts

```diff
@@ -270,7 +270,14 @@ export class ExtensionInstance<TConfiguration extends BaseConfigType = BaseConfi
   async bundleConfig({identifiers, token, apiKey, unifiedDeployment}: ExtensionBundleConfigOptions) {
     const configValue = await this.deployConfig({apiKey, token, unifiedDeployment})
     if (!configValue) return undefined
-    return {uuid: identifiers.extensions[this.localIdentifier]!, config: JSON.stringify(configValue), context: ''}
+
+    const contextValue = (configValue.handle as string) || ''
+
+    return {
+      uuid: identifiers.extensions[this.localIdentifier]!,
+      config: JSON.stringify(configValue),
+      context: contextValue,
+    }
```

Comment:
We should send an stringified json object instead of a string, to make it future proof in case we want to add more data here later.

Reply:
this is a temporary measure so I think we can use string for now. When the proper `handle` implementation is out, this will be gone.